### PR TITLE
Adding email sending functionality and uptade of readme.md file

### PR DESCRIPTION
**Pull request by monwd**

### CODE REVIEW

File: html_generator.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import meal_planner_classes
+
+
+class Meals:
```
Comment (milkisbad) [inline html_generator.py:4]: this is probably unnecessary, but for now change the name

File: html_generator.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          {self.generate_table_header(column_names)}
+          {self.generate_content(rows)}
+          </table>
+          </body>
+          </html>
+           """
+        return table_html
+
+
+def main_generator():
```
Comment (milkisbad) [inline html_generator.py:77]: change the name

File: meal_planner_classes.py [hunk 3]
```diff
@@ -87,68 +53,35 @@ def __init__(self, num_of_days):
...
-        num_of_days = int(input("For how many days do you want to plan meals?  \n"))
+        num_of_days = int(input("For how many days do you want to plan meals?   \n"))
         return num_of_days
 
     except ValueError:
         raise ValueError("Provide integer value")
 
 
-if __name__ == "__main__":
+def main():
```
Comment (milkisbad) [inline meal_planner_classes.py:77]: change the name

File: sending_emails.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    # Connect to Gmail's SMTP Outgoing Mail server with such context
+    with smtplib.SMTP_SSL("smtp.gmail.com", 465, context=context) as server:
+        server.login(email_from, password)  # Provide Gmail's login information
+        server.sendmail(email_from, email_to, email_message.as_string())
+
+    print(f"Meal planner has been successfully sent to: {email_to}!")
+
+
+def main_sending_email():
```
Comment (milkisbad) [inline sending_emails.py:38]: change the name

File: main.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+from sending_emails import main_sending_email
+
+
```
Comment (milkisbad) [inline main.py:3]: in here place a function looking like this
`def main(generate_html_table: bool, send_email: bool)`
and use it in the if __name__...

File: meal_planner_classes.py [hunk 6]
```diff
@@ -87,49 +85,48 @@ def __init__(self, num_of_days):
...
+          </tr>
+          </tbody>
+          </table>
+
+           """
         return table_html
 
     def display_html(self, html_str):
-        display(HTML(html_str))
+        display (HTML(html_str))
```
[the remark this answers is not in the document]
Comment (monwd) [reply meal_planner_classes.py]: this is outdated

File: meal_planner_classes.py [hunk 7]
```diff
@@ -138,17 +135,27 @@ def __str__(self):
...
     df_recipes = pd.read_csv("recipes_base.csv", index_col=0, sep=";")
+    cookbook = Cookbook(df_recipes=df_recipes)
     num_of_days = get_input()
     w = WeeklyPlan(num_of_days)
-    w.draw_meal_plan()
+    w.draw_meal_plan(cookbook)
+    # html_str = HtmlGenerator(w.list_of_day_plans).create_html_table()
+    # display(HTML(html_str))
+    # w.display_html(html_str)
+    # print(html_str)
```
[the remark this answers is not in the document]
Comment (monwd) [reply meal_planner_classes.py]: this is outdated

**Pull request merged**